cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -183,6 +193,12 @@ class LeadImpl extends ServerImpl with Lead
 
   }
 
+  /* Returns true if LDAP Security is Enabled */
```

Comment:
This was quickly done initially. Could be refactored to avoid code duplication. Do see Misc.isSecurityEnabled() which too has this code.